examples/dftb_uv_spectrum/train_smooth_uv_spectrum.py

```diff
@@ -151,9 +151,8 @@ def __init__(self, dirpath, dftb_node_types, var_config, dist=False, sampling=No
                 extract_path = extract_tar_file(fullpath, tmpfs)
                 mdirs = get_mol_dir_list(extract_path)
                 assert len(mdirs) > 0, f"No molecules found in tar file {fullpath} extracted at {extract_path}"
-                for mdir in mdirs:
+                for mdir in iterate_tqdm(mdirs, verbosity_level=2, desc="Processing"):
```

Comment:
Is line 155 redundant? `data_object` is being set on line 156 as well.

Reply:
@kshitij-v-mehta you are right. It is duplicated and that is why I am getting errors with atomicdescriptors. I think using `transform_input_to_data_object_base` is the right way. I will fix.

Reply:
I found `dftb_to_graph` is the correct way. I switched.